### DISCUSSION TITLE
fabric-api is missing!

### DISCUSSION THREAD

**Discussion by Sauliuxz** [FabricMC/fabric-api#3245] [Mod Dev Support]
### Description

![image](https://github.com/FabricMC/fabric/assets/78691577/38020599-eb01-4438-9637-7d095acc6e61)

Hi everyone, I just setted up my first mod, nothing yet, but I get this error. Where should I install fabric-api? I tried to put it in .minecraft/mods but when I compile, I still get the same error. Thanks in advance.

**Comment by ZhillerDev** [comment]
I have the same error like this
I try to use different version of fabric but it still didn't work

**Reply by haykam821** [reply]
Does [this comment](https://github.com/orgs/FabricMC/discussions/3245#discussioncomment-6698517) help with your issue?

**Reply by ZhillerDev** [reply]
Yes, thanks a lot, I fixed this error！

**Comment by haykam821** [comment]
What version of Minecraft are you trying to develop a mod for? Would it be possible to share your `build.gradle` and `gradle.properties` files? The information you've provided isn't enough to diagnose the issue.

If you're developing for an older Minecraft version, you may be running into FabricMC/fabricmc.net#49. For older Minecraft versions, the primary mod ID used for Fabric API is `fabric` instead of `fabric-api`, so you can update the `depends` entry in the `fabric.mod.json` file to use the old mod ID.

**Reply by ZhillerDev** [reply]
Thank you so much for your help!

I tried to modify it according to this method, no error was reported, and it can be run!

But I am using version 1.19, does this problem also occur in higher versions?
